### DISCUSSION TITLE
Aprendiendo GitHub

### DISCUSSION THREAD

**Discussion by cuantocarlos** [PucelaBits/websegura#59] [Preguntas y respuestas] [answered]
![imagen](https://user-images.githubusercontent.com/78327050/106790664-2640d080-6654-11eb-9d92-e429e38a6c4a.png)
Estoy viendo tutoriales en internet y recomiendan añadir los repositorios directamente a Atom y me parece lo + cómodo. 
Puede ser que no se pueda por que primero hay que crear un Pull requests (que tampoco conozco muy bien como va aún)?

**Comment by palmerabollo** [comment]
No uso Atom (me va más VS Code), pero en github puedes clonar con varios protocolos, prueba con git@github.com:PucelaBits/websegura.git, que creo que puede ser eso. Lo puedes ver desplegando el botón "Code". Avisa si te da algún problema.

![Screenshot 2021-02-03 at 19 54 40](https://user-images.githubusercontent.com/673420/106794981-cf3dfa00-6659-11eb-8f0e-3f679617b4d6.png)

**Comment by nukeador** [comment, marked as answer]
Creo que primero tienes que crear un fork de este repo, y es tu fork el que configuras localmente (ya que es donde tienes permisos para hacer commits).

Cuando haces cambios en tu fork (commit) y los subes (push(, github te da una URL para generar un PR desde tu fork al principal.

Un saludo.

**Reply by cuantocarlos** [reply]
Interesante, lo investigo
